ocaml/xapi-cli-server/cli_operations.ml

```diff
@@ -4765,9 +4765,17 @@ let host_all_editions printer rpc session_id params =
   printer (Cli_printer.PList editions)
 
 let host_evacuate printer rpc session_id params =
+  let network =
```

Comment:
Could be written as:
```
let network = List.assoc_opt "network" params |> Option.fold ~none:Ref.null ~some:Ref.of_string
```

Reply:
Yes, it's simpler!

Reply:
Was there a reason not to change the code?

Reply:
I forgot ^^'
Done now :)